metrics/prometheus.go

```diff
@@ -0,0 +1,92 @@
+package metrics
+
+import (
+	"strings"
+	"sync/atomic"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func NewPrometheusRegistry(registry *prometheus.Registry, service string) Registry {
+	return &PrometheusRegistry{registry, service}
+}
+
+type PrometheusRegistry struct {
+	registry *prometheus.Registry
+	service  string
+}
```

Comment:
nit: Just a personal preference. I think that definition of constructor is looking better after definition of type that this constructor instantiate. What do you think about swapping type definition and func definition with each other?

Reply:
It's fine for me. Also it seems we are able to make `PrometheusRegistry` private, the same is for `GraphiteRegistry`. 

PS I think we need some kind of a linter to fix the style instead of these kind of discussions and manual reformat of the code.

Reply:
Done